### DISCUSSION TITLE
Possible fix for the Tweakscale Hierarchy bug

### DISCUSSION THREAD

**Discussion by Starcatcher2009** [TweakScale/TweakScale#304] [Support]
I figured out by running KSP as admin right in its directory, with no shortcut, does not bug Tweakscale with any assembly loading bug. Perhaps you can include that as a possible fix in the wiki?

**Comment by Lisias** [comment]
Wow… This is serious?

Can you send me a KSP.log where TweakScale borks due the assembly loading bug but then works under Admin?

Running games as Admin is a terrible security risk, by the way. There're supply chains attacks happening on the game industry, and I consider a miracle we don't have any on KSP yet.

I intend to use this information to **properly** diagnose this crap and then try to do some preemptive action to prevent it, or perhaps an automated fix?
